concepticondata/conceptlists/Luangthongkum-2019-341.tsv

```diff
@@ -0,0 +1,342 @@
+Luangthongkum-2019-341-168	168	linear measure of about half a yard (closed-hand)		
+Luangthongkum-2019-341-169	169	linear measure of about half a yard (open-hand)		
+Luangthongkum-2019-341-170	170	linear measure of about two yards		
+Luangthongkum-2019-341-171	171	listen to	1608	LISTEN
+Luangthongkum-2019-341-172	172	live, stay		
+Luangthongkum-2019-341-173	173	liver	1224	LIVER
+Luangthongkum-2019-341-174	174	long (of time)	418	FOR A LONG TIME
+Luangthongkum-2019-341-175	175	long (of length), tall, high		
+Luangthongkum-2019-341-176	176	look after (things)		
+Luangthongkum-2019-341-177	177	loris		
+Luangthongkum-2019-341-178	178	louse (chicken~), insect	1392	LOUSE
+Luangthongkum-2019-341-179	179	louse (head~)	310	HEAD LOUSE
+Luangthongkum-2019-341-180	180	lung	688	LUNG
+Luangthongkum-2019-341-181	181	maggot		
+Luangthongkum-2019-341-182	182	male (of animals), father		
+Luangthongkum-2019-341-183	183	man, male	1554	MAN
+Luangthongkum-2019-341-184	184	mango	2398	MANGO
+Luangthongkum-2019-341-185	185	many, much	1198	MANY
+Luangthongkum-2019-341-186	186	mat	195	MAT
+Luangthongkum-2019-341-187	187	milipede		
+Luangthongkum-2019-341-188	188	mole	1907	MOLE
+Luangthongkum-2019-341-189	189	money	1559	MONEY
+Luangthongkum-2019-341-190	190	monitor lizard (land~)	3017	MONITOR LIZARD
+Luangthongkum-2019-341-191	191	monitor lizard (water~)		
+Luangthongkum-2019-341-192	192	monkey	1350	MONKEY
+Luangthongkum-2019-341-193	193	moon, month	2463	MOON OR MONTH
+Luangthongkum-2019-341-194	194	mother, female (of animals)		
+Luangthongkum-2019-341-195	195	mouse, rat	2139	MUROID (MOUSE OR RAT)
+Luangthongkum-2019-341-196	196	mynah		
+Luangthongkum-2019-341-197	197	nail		
+Luangthongkum-2019-341-198	198	name	1405	NAME
+Luangthongkum-2019-341-199	199	narrow	1267	NARROW
+Luangthongkum-2019-341-200	200	navel, umbilical cord		
+Luangthongkum-2019-341-201	201	neck, cheek		
+Luangthongkum-2019-341-202	202	new	1231	NEW
+Luangthongkum-2019-341-203	203	nine	1483	NINE
+Luangthongkum-2019-341-204	204	obtain, receive, take, get	694	GET
+Luangthongkum-2019-341-205	205	old (of humans)	2112	OLD (AGED)
+Luangthongkum-2019-341-206	206	old (of things)	2113	OLD (USED)
+Luangthongkum-2019-341-207	207	one	1493	ONE
+Luangthongkum-2019-341-208	208	open (the mouth)	3437	OPEN (MOUTH)
+Luangthongkum-2019-341-209	209	ox, bull, cow	3044	BOVINE
+Luangthongkum-2019-341-210	210	palm (of the hand)	1183	PALM OF HAND
+Luangthongkum-2019-341-211	211	pangolin, anteater		
+Luangthongkum-2019-341-212	212	parrot, parakeet		
+Luangthongkum-2019-341-213	213	path, trail, way	2252	PATH
+Luangthongkum-2019-341-214	214	peacock	3382	PEACOCK
+Luangthongkum-2019-341-215	215	pen, coop	3401	HENHOUSE
+Luangthongkum-2019-341-216	216	penis	1222	PENIS
+Luangthongkum-2019-341-217	217	pheasant	3060	PHEASANT
+Luangthongkum-2019-341-218	218	pig	1337	PIG
+Luangthongkum-2019-341-219	219	pile		
+Luangthongkum-2019-341-220	220	pointed (as of needles), sharp point		
```

Comment:
372 POINTED?

Reply:
I didn't want to map this gloss because its ontological category refers both to the object and its property (POINTED 372 and POINT (POINTED END) 3255).